examples/telegram/interfaces/polling.py

```diff
@@ -0,0 +1,59 @@
+# %% [markdown]
+"""
+# 1. Polling
+
+The following example shows how to integrate your bot with the Pipeline API.
+
+Assuming that you already have a script that you need to deploy, all you need to do
+is to instantiate a TelegramMessenger and pass it to an `Interface` class.
+This class uses `PollingTelegramInterface` for local deployment without webhooks.
+"""
+
+
+# %%
+import os
+
+from dff.connectors.messenger.telegram.interface import PollingTelegramInterface, TelegramMessenger
+from dff.core.pipeline import Pipeline
+
+from dff.utils.testing.common import is_interactive_mode, run_interactive_mode
+from dff.utils.testing.toy_script import TOY_SCRIPT
+
+
+# %% [markdown]
+"""
+Like Telebot, TelegramMessenger only requires a token to run.
+However, all parameters from the Telebot class can be passed as keyword arguments.
+"""
+
+
+# %%
+messenger = TelegramMessenger(os.getenv("TG_BOT_TOKEN", "SOMETOKEN"))
+
+
+# %% [markdown]
+"""
+You only need an instance of `TelegramMessenger` for polling.
+"""
+
+
+# %%
+interface = PollingTelegramInterface(messenger=messenger)
+
+
+# %%
+pipeline = Pipeline.from_script(
+    script=TOY_SCRIPT,  # Actor script object, defined in `.utils` module.
+    start_label=("greeting_flow", "start_node"),
+    fallback_label=("greeting_flow", "fallback_node"),
+    context_storage=dict(),
+    messenger_interface=interface,  # The interface can be passed as a pipeline argument.
+)
+
+if __name__ == "__main__":
```

Comment:
Shouldn't we add `check_happy_path(pipeline, happy_path)` like in other examples?
Something like:
```python
if __name__ == "__main__":
    if not os.getenv("TG_BOT_TOKEN"):
        print("`TG_BOT_TOKEN` variable needs to be set to use TelegramInterface.")
    else:
        check_happy_path(pipeline, happy_path)
        if is_interactive_mode():
            run_interactive_mode(pipeline)  # run in an interactive shell
```